firmware/tunerstudio/rusefi.input

```diff
@@ -2003,16 +2003,16 @@ cmd_stop_engine              = "w\x00\x99\x00\x00"
 		field = "ADC #3",								fsioAdc3
 		field = "ADC #4",								fsioAdc4
 				
-	dialog = veTableDialogNorth
-		field = "Volumetric Efficiency is used to calculate fuel in Speed Density mode"
-		field = "http://rusefi.com/fuel"
+	help = veTableDialogHelp, "Volumetric Efficiency"
+        text = "Volumetric Efficiency is used to calculate fuel in Speed Density mode"
+        webHelp = "http://rusefi.com/fuel"
```

Comment:
This creates help menu item filled with our info, and web-link to the site also. Nice and clean!